awx/ui_next/src/screens/Setting/shared/RevertButton.jsx

```diff
@@ -45,7 +45,7 @@ function RevertButton({ i18n, id, defaultValue, isDisabled = false }) {
       <ButtonWrapper>
         <Button
           aria-label={isRevertable ? i18n._(t`Revert`) : i18n._(t`Undo`)}
-          data-cy={`${id}-revert`}
+          data-ouia-id={`${id}-revert`}
```

Comment:
it should be `ouiaId`:
```js
ouiaId={`${id}-revert`}
```

Reply:
Thanks! I can confirm that adding `ouiaId` to the Revert Button transforms into the following additional attributes:
* data-ouia-component-type="PF4/Button" 
* data-ouia-safe="true" 
* data-ouia-component-id="TOWER_URL_BASE-revert"